src/schemas/ncs-benefits/schema.js

```diff
@@ -0,0 +1,49 @@
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+import _ from 'lodash';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: "APPLICATION FOR VA EDUCATION BENEFITS UNDER THE NATIONAL CALL TO SERVICE (NCS) PROGRAM (22-1990N)",
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'educationType',
+    'dateRange'
+  ]),
+  properties: {
```

Comment:
There's a question on the GH [issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/632), "What is the source of these funds?" that I don't see.

Also, "Are you getting, or do you expect to get any money (including, but not limited to, federal tuition assistance) from the Armed Forces or public health services for any part of your coursework or training?" from [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/633).

And preferredContactMethod ([here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/634))